### ISSUE TITLE
[Rewrite] Supported Operating Systems and PHP Versions with Repositories

### ISSUE THREAD

**Issue by SimonHoenscheid** [voxpupuli/puppet-php#604]
This Issue contains a Matrix of supported operation systems and PHP versions with additional repositories (if any)

**Comment by igalic** [comment]
Does Debian 8 	NO (ELTS), mean that this OS will be supported longer than we think it will?
cuz we've started removing it from modulesync

**Comment by SimonHoenscheid** [comment]
@igalic yes! It's documented in the Debian wiki. But, I am not sure what this means for Voxpupuli modules

**Comment by igalic** [comment]
judging from the info here, https://wiki.debian.org/LTS/Extended i don't think we can afford to test on an ELTS release, so it feels appropriate to deprecate it as soon as it hits LTS EOL

**Comment by SimonHoenscheid** [comment]
@igalic removed Debian 8

**Comment by avitacco** [comment]
Release candidates for PHP 8 are in the Sury repo.
